quickcomm/models.py

```diff
@@ -30,16 +175,60 @@ class Author(models.Model):
     # Taken on: Feb 6, 2023
 
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    host = models.URLField(validators=[URLValidator])
+    # FIXME the user is only null if the author is remote. This should be
```

Comment:
don't we still need a user attribute here? just make it so it can be null for now and i'll take care of it in my update